addons/sys_gestures/XEH_postInit.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+if (is3DEN || {!hasInterface || {!(GVAR(gesturesEnabled))}}) exitWith {};
+if (!isClass (configFile >> "CfgPatches" >> "ace_common")) exitWith {}; // No ACE exit
+
+GVAR(vestRadioArr) = call compile (GVAR(vestRadios));
+GVAR(headsetRadioArr) = call compile (GVAR(headsetRadios));
+
+if (GVAR(stopADS)) then {
+    GVAR(disallowedViews) = ["GROUP"];
+};
+
+["acre_startedSpeaking", {
+    params ["_unit", "_onRadio", "_radio"];
+
+    if (!_onRadio ||
```

Comment:
onRadio is not always a Bool it can also be a Number!

Reply:
What! Why?!

Reply:
ask @diwako :D he loved this issue. ref: https://github.com/diwako/diwako_dui/commit/0965811b3afd9f8491bcd63abd34dbfbbc7d2a00

Reply:
IIRC it is only for `startedRemoteSpeaking` the case.

Reply:
oh I thought it is always

Reply:
I'd answer if I could, better not to ask. I streamlined it a tiny bit in last release, but backwards compatibility didn't let us go further.

Reply:
Docs list `startedSpeak` as onRadio and `remoteStartedSpeaking` as speakingType so guess this is safe.